ZAPD/ZTexture.cpp

```diff
@@ -820,54 +823,33 @@ fs::path ZTexture::GetPoolOutPath(const fs::path& defaultValue)
 	return defaultValue;
 }
 
-TextureType ZTexture::GetTextureTypeFromString(std::string str)
+TextureType ZTexture::GetTextureTypeFromString(std::string type)
 {
-	TextureType texType = TextureType::Error;
+	static const std::map<std::string, TextureType> ZTextureTypeDictionary = {
```

Comment:
I _think_ `unsorted_map` is faster for this kind of thing compared to plain `map`
From cppreference:
> Unordered associative containers implement unsorted (hashed) data structures that can be quickly searched (O(1) amortized, O(n) worst-case complexity). 

<https://en.cppreference.com/w/cpp/container>

Reply:
I changed this where possible. Unfortunately `std::hash` can't handle fancy stuff like pairs or tuples without writing a whole new function to hash, so I couldn't change the ZDisplayListTexture one.